roxie/roxiemem/roxiemem.cpp

```diff
@@ -461,6 +537,69 @@ extern void releaseRoxieHeap()
     }
 }
 
+extern void notifyAllUnusedRoxieMem()
+{
+    char *blockStart = 0;
+    char *blockEnd = 0;
```

Comment:
picky suggestion.  Code would be *slightly* simpler if memsize_t blockLength was maintained rather than blockEnd.